### PR TITLE
EJ2-Documentation-851726-Ug-Documentation-For-Selection

### PR DESCRIPTION
**Pull request by harinikrh**

### CODE REVIEW

**Review by VigneshNatarajan27 — CHANGES_REQUESTED** [review]
highlight clearColumnSelection method and link it to API documentation.

**Comment by harinikrh** [comment]
> highlight clearColumnSelection method and link it to API documentation.

Linked the API

File: ej2-javascript/code-snippet/grid/Selection-grid-toggle/index.js [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+    { field: 'OrderID', headerText: 'Order ID', textAlign: 'Right', width: 120},
+    { field: 'CustomerID', headerText: 'Customer ID', width: 150 },
+    { field: 'ShipCity', headerText: 'Ship City', width: 150 },
+    { field: 'ShipName', headerText: 'Ship Name', width: 150 },
+  ],
+  height: 315,
+});
+grid.appendTo('#Grid');
+
+var enableToggleSelection = true;
```
Comment (MaithiliyK) [inline ej2-javascript/code-snippet/grid/Selection-grid-toggle/index.js:14]: what is the use of defining this variable here. Its value is not changed anywhere in the sample. Is it needed?

File: ej2-javascript/code-snippet/grid/Selection-grid-toggle/index.js [hunk 2]
```diff
@@ -0,0 +1,43 @@
...
+
+var toggle = new ej.buttons.Switch({
+  change: toggleColumnSelection,
```
Comment (MaithiliyK) [inline ej2-javascript/code-snippet/grid/Selection-grid-toggle/index.js:17]: in change event itself you can get the argument regarding the checked status?

**Review by MaithiliyK — COMMENTED** [review]
address my review comments

**Pull request merged**